candidpartners/terraform-provider-candidaws
# CFS-7074 - Terraform Provider AWS - QuickSight Integration

To test:
`docker run --rm -ti -v $PWD:/code golang:1.16-buster bash -c "cd /code; go build -o bin/linux_amd64/terraform-provider-candidaws.exe --mod=vendor"`

The above build command (via Docker) should exit cleanly and produce the desired Terraform provider binary.

Notes:
Additional testing possible once INC000401314336 is resolved.

Sample Terraform using provider:
```
terraform {
  required_version = ">= 0.12.31"
  required_providers {
    aws       = ">= 3.3.0"
    candidaws = "1.1.6"
  }
}

provider candidaws {
  region = "us-gov-west-1"
  alias = "test"
}
provider aws {
  region = "us-gov-west-1"
}

resource "aws_quicksight_user" "example" {
  user_name     = "USER"
  email         = "USER@example.com"
  identity_type = "QUICKSIGHT"
  user_role     = "ADMIN"
}

resource "aws_quicksight_group" "example" {
  group_name = "group-test"
}

resource "aws_quicksight_group_membership" "example" {
  provider = candidaws.test
  member_name = "USER"
  group_name = "group-test"
}

resource "aws_quicksight_namespace" "example" {
  provider = candidaws.test
  identity_store = "QUICKSIGHT"
  namespace = "namespace-test"
}

resource "aws_quicksight_namespace" "example2" {
  provider = candidaws.test
  identity_store = "QUICKSIGHT"
  namespace = "namespace-test2"
  tags = {
    Name = "test-tag"
  }
}
```
